orchagent/copporch.cpp

```diff
@@ -451,14 +451,13 @@ bool CoppOrch::removeGenetlinkHostIf(string trap_group_name)
     return true;
 }
 
-task_process_status CoppOrch::processCoppRule(Consumer& consumer)
+task_process_status CoppOrch::processCoppRule(Consumer& consumer, SyncMap::iterator it)
```

Comment:
It seems this function only need the "tuple" as input, why we don't change it to:
task_process_status CoppOrch::processCoppRule(KeyOpFieldsValuesTuple tuple)

to make it clearer?